Snake/BusinessLayer/FoodFolder/Food.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace BusinessLayer.FoodFolder
+{
+    public class Food
```

Comment:
Can I ever create this class? i.e. Food f = new Food();
I suspect not - so we should make it an [Abstract Class](https://msdn.microsoft.com/en-us/library/ms173150.aspx)